src/test/groovy/MainServerControllerTest.groovy

```diff
@@ -0,0 +1,12 @@
+import spock.lang.Specification
+
+/**
+ * Created by webonise on 14/9/16.
+ */
+class MainServerControllerTest extends Specification {
+
+    def 'test something very basic'(){
```

Comment:
Yes I will take care of this. The intention of using this basic test code was just to maintain the project structure. With this test code empty test directory wouldn't have been pushed.